packages/css/src/components/card/card.scss

```diff
@@ -16,12 +15,24 @@
     outline-style: auto;
     outline-width: 0.0625rem;
   }
+
+  & > .ams-heading:has(+ *) {
+    margin-block-end: var(--ams-space-s);
```

Comment:
Please add intermediate component tokens (e.g. `--ams-card-heading-margin-block-end`) so that teams using our components can choose their own spacing values.